styles.css

```diff
@@ -237,6 +248,14 @@ hr {
   background-color: rgb(40, 41, 49);
 }
 
+.about-program {
+  border-top: 2px solid black;
+  border-bottom: 2px solid black;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
 .programi {
```

Comment:
Great design @godybuoy 🤝

- [ ] Kindly ensure the cards on your main program section do not have a vertical scroll. This will make your design a replica of the design provided.
- Your design
![Your-main-prgrams](https://user-images.githubusercontent.com/63917725/235324748-9b506fba-7d4b-4bc0-be32-25a257f5e632.PNG)
- Given design
![Given-main programs](https://user-images.githubusercontent.com/63917725/235324756-3f6e7c88-640b-4fbb-b9ad-ce361c7ccb0f.PNG)